File: DisplaBackend/DisplaBackend/Models/PrecioServicioCliente.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class PrecioServicioCliente
    {
        public int Id { get; set; }
        public int IdPrecioServicio { get; set; }
        public int IdCliente { get; set; }
        public double? Descuento { get; set; }
        public bool? Especial { get; set; }

        public virtual Cliente IdClienteNavigation { get; set; }
        public virtual PrecioServicio IdPrecioServicioNavigation { get; set; }
    }
}

File: DisplaFrontend/src/app/model/precioServicioCliente.ts
import { PrecioArticulo } from './precioArticulo';
import { Cliente } from './cliente';
import { PrecioServicio } from './precioServicio';

export interface PrecioServicioCliente {
    Id: number,
    IdPrecioServicio: number,
    IdPrecioServicioNavigation: PrecioServicio
    IdCliente: number,
    IdClienteNavigation: Cliente,
    Descuento: number,
    Especial: boolean
}
